catroid/src/org/catrobat/catroid/content/Sprite.java

```diff
@@ -302,6 +304,32 @@ public Sprite clone() {
 		return cloneSprite;
 	}
 
+	public Sprite cloneForCloneBrick() {
+		final Sprite cloneSprite = new Sprite();
+
+		cloneSprite.setName(this.getName() + "-c" + StageActivity.getAndIncrementNumberOfClonedSprites());
+		cloneSprite.isClone = true;
+		cloneSprite.isBackpackObject = false;
+
+		//cloneSprite.lookList = this.lookList;
+		cloneSprite.soundList = this.soundList;
```

Comment:
Ahh I see. You are partially right, the look needs to be cloned, but cloning lookDatas is not needed. But as you said, this is not costly, so I would say you can leave it that way since otherwise you would need either another method or more code which is no good ;)